virt/virt-4-12-release-notes.adoc

```diff
@@ -182,9 +150,29 @@ Deprecated features are included in the current release and supported. However,
 [id="virt-4-12-removed"]
 === Removed features
 
+Removed features are not supported in the current release.
```

Comment:
Are we supposed to keep the "Removed features" from the previous release? For example, the NMO RN talks about tasks that the user needs to perform before updating to 4.11. Keeping these notes in 4.12 might cause some confusion.

Reply:
Not usually, but we made the decision with two this time to keep for one more release in case someone hasn't upgraded from 4.10 yet, since it's significant impact.